crates/forge/tests/e2e/running.rs

```diff
@@ -1003,3 +1004,30 @@ fn incompatible_snforge_std_version_warning() {
         "}
     );
 }
+
+#[test]
+fn detailed_resources_flag() {
+    let temp = setup_package("complex_package");
```

Comment:
Do we need another cairo package to test that? Cannot we reuse an already existing package?

Reply:
Agree on that with Piotr